tests/data/assets.sh

```diff
@@ -29,8 +11,8 @@ RUSTDATADIR="crates/dekoder/tests/data/"
 # download from the server
 curl -s -C - -o "./${PYDATADIR}v0.13.tar" "${URL}v0.13.tar"
 curl -s -C - -o "./${PYDATADIR}v0.14.tar" "${URL}v0.14.tar"
-curl -s -C - -o "./${PYDATADIR}v0.0.tar" "${URL}v0.0.tar"
+curl -s -C - -o "./${PYDATADIR}v0.15.tar" "${URL}v0.15.tar"
```

Comment:
let's rename the files both locally and remotely (i.e. on the server):

- `v0.13.tar` -> `v1-0.13.tar`
- `v0.14.tar` -> `v1-0.14.tar`
- `v0.15.tar` -> `v3.tar`

for the usual reason we have to fake v1, but henceforth the files correspond to a given data version and not a API version